.travis.yml

```diff
@@ -1,29 +1,19 @@
+dist: bionic
 language: python
-dist: bionic   # required for Python >= 3.7
 cache: pip
-
-# Environment changes have to be manually synced with 'tox.ini'.
-# See: https://github.com/travis-ci/travis-ci/issues/3024
-
-python:
-  - "3.6"
-  - "3.7"
-
 matrix:
   include:
-    - python: 3.5
+    - python: "3.5"
       dist: xenial  # Bionic has no Python 3.5
-
+    - python: "3.6"
+    - python: "3.7"
     # Disabled, see https://github.com/sybrenstuvel/python-rsa/issues/131
     #- python: pypy3.5
     #  dist: xenial  # Bionic has no Python 3.5
-
+before_install: pip install --upgrade pip
```

Comment:
Why is this necessary?

Reply:
Because Travis provides an older version of pip and pip is going thru rapid innovation these daze.  There is hope that pip will add a real dependency resolver in the near term.

Reply:
Would be good to add a comment to this effect, including which version of Pip is now installed by default.